src/main/resources/digital/slovensko/autogram/ui/gui/password-dialog.fxml

```diff
@@ -13,6 +13,7 @@
 
     <VBox fx:id="formGroup" styleClass="autogram-form-group">
         <TextFlow><Text fx:id="question" styleClass="autogram-heading-l" VBox.vgrow="ALWAYS" text="Aký je kód k&#160;úložisku certifikátov?"/></TextFlow>
+        <TextFlow><Text fx:id="description" styleClass="autogram-description" /></TextFlow>
```

Comment:
Týmto si zrejme nechtiac pridal vertikálne prázdne miesto do toho controllera. Vieš tomu TextFlow nastaviť visible a managed na false. Napr ako tu: https://github.com/slovensko-digital/autogram/blob/main/src/main/resources/digital/slovensko/autogram/ui/gui/signing-dialog.fxml#L25 (visible hovorí, či je element vidno, managed hovorí, či má zaberať mmiesto akokeby - ak dáš iba visible false, TextFlow nebude vidno, ale miesto bude aj tak zaberať)

A potom v controlleri to zapneš, iba keď description nie je null.